tests/functional/testOnboarding.js

```diff
@@ -68,13 +59,17 @@ describe('Onboarding', function() {
      });
 
   it('Sign up button opens auth flow', async () => {
+    // must wait for animation
+    await vpn.wait()
     await vpn.waitForQueryAndClick(
         queries.screenInitialize.SIGN_UP_BUTTON.visible());
     await vpn.waitForQuery(
         queries.screenAuthenticationInApp.AUTH_START_TEXT_INPUT.visible());
   });
 
   it('Already a subscriber? opens auth flow', async () => {
+    // must wait for animation
+    await vpn.wait()
```

Comment:
Which animation you are talking about? The Lottie? If that, we don't need to wait for it.
If you are waiting for the global loader to load the current screen, use:
`await vpn.waitForQuery(queries.global.SCREEN_LOADER.ready());`

Reply:
ive tried this wait for screen loader, it did not work.

Reply:
It needs to be debugged.